tiles/rooms/floor/staircase.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1"
+	 id="svg2" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cc="http://creativecommons.org/ns#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:svg="http://www.w3.org/2000/svg" xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd" xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape" inkscape:version="0.48.5 r10040" sodipodi:docname="caro.svg"
+	 xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="430.543px"
+	 height="325.295px" viewBox="0 0 430.543 325.295" enable-background="new 0 0 430.543 325.295" xml:space="preserve">
+<defs>
+	
+	
+		<inkscape:perspective  sodipodi:type="inkscape:persp3d" inkscape:vp_x="0 : 526.18109 : 1" inkscape:vp_y="0 : 1000 : 0" inkscape:vp_z="744.09448 : 526.18109 : 1" id="perspective2985" inkscape:persp3d-origin="372.04724 : 350.78739 : 1">
+		</inkscape:perspective>
+</defs>
+<sodipodi:namedview  units="mm" pagecolor="#ffffff" bordercolor="#666666" borderopacity="1.0" inkscape:zoom="1.979899" inkscape:cx="169.11117" inkscape:cy="150.5455" showgrid="false" showguides="true" fit-margin-top="0" fit-margin-left="0" id="base" inkscape:pageopacity="0.0" inkscape:pageshadow="2" inkscape:document-units="mm" inkscape:current-layer="layer1" inkscape:object-nodes="true" inkscape:window-width="1920" inkscape:window-height="1036" inkscape:window-x="4" fit-margin-bottom="0" inkscape:window-y="44" inkscape:window-maximized="0" inkscape:guide-bbox="true" fit-margin-right="0">
+	</sodipodi:namedview>
+<g>
+	<defs>
+		<path id="path3908" inkscape:connector-curvature="0" d="M420.733,202.206L215.272,79.612L9.812,202.206l205.461,122.595
+			L420.733,202.206z"/>
+	</defs>
+	<clipPath id="path3908_1_">
+		<use xlink:href="#path3908"  overflow="visible"/>
+	</clipPath>
+	<g transform="matrix(1 0 0 1 9.536743e-007 -2.384186e-007)" clip-path="url(#path3908_1_)">
+		
+			<image overflow="visible" width="576" height="576" xlink:href="../../../../Desktop/staircase.png"  transform="matrix(0.7352 0.0147 -0.0147 0.7352 11.2407 -2.3457)">
```

Comment:
just saw this: ../../../../Desktop/staircase.png will not work here. Could you put the staircase.png next to this file and change this to ./staircase.png?

Reply:
I think it is a good idea not to embed since it decreases the size.

Reply:
> Could you put the staircase.png next to this file and change this to ./staircase.png?

Seems like it didn't work, how do I keep IMAGE as svg without embedding?